shell/packages/sandstorm-db/db.js

```diff
@@ -843,6 +894,30 @@ _.extend(SandstormDb.prototype, {
     return user && Plans.findOne(user.plan || "free");
   },
 
+  getMyReferralBonus: function(user) {
```

Comment:
This function taking a user parameter feels weird to me. The name of the function is get**My**ReferralBonus, which implies that it only works for the current user. Also, the logic to  ignore the parameter client-side seems messy and inconsistent to me.